api/config/12-db-deployment.yaml

```diff
@@ -57,6 +57,16 @@ spec:
           volumeMounts:
             - name: db
               mountPath: /var/lib/postgresql/data
+          readinessProbe:
+            exec:
+              command: [psql, -w, -U, postgres, -d, $(POSTGRES_DB), -c, "SELECT 1"]
```

Comment:
does this work? what does $(POSTGRES_DB) eval to?

Reply:
`$(POSTGRES_DB)` refers value from env variable defined above

Reply:
```  
┆ ┆ ┆ ┆ ┆ - bash
  ┆ ┆ ┆ ┆ ┆ - -c
  ┆ ┆ ┆ ┆ ┆ - psql -w -U ${POSTGRES_USER} -d ${POSTGRES_DB} -c 'select 1 as query;'
  ```
works with env

Reply:
It works. Thank you :)